docs/modules/ROOT/pages/openshift-service-mesh/index.adoc

```diff
@@ -0,0 +1,40 @@
+= OpenShift Service Mesh
+
+RedHat OpenShift Service Mesh is a tool for traffic management, tracing, service authentication and access policy enforcement within the OpenShift cluster. It's based on Istio, Kiali, and Jaeger.
```

Comment:
Generally we try to keep our documentation sources to one line per sentence. This tends to make diffs clearer when making changes later on.